src/components/molecules/Card/Card.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// A T O M S
+import { Icon, IconName, IconSize } from '../../atoms/Icon/Icon'
+
+export interface CardProps {
+  className?: string
+  footer?: React.ReactNode
+  icon?: IconName
+  iconSize?: IconSize
+  imageUrl?: string
+  imageMargin?: boolean
+  linkContainerMinHeight?: string
+  linkUrl?: string
+  style?: React.CSSProperties
+  title?: string
+}
+
+export const Card: React.FC<CardProps> = ({
+  children,
+  className,
+  footer,
+  icon,
+  iconSize,
+  imageUrl,
+  imageMargin = false,
+  linkContainerMinHeight = '11.25rem',
+  linkUrl,
+  style,
+  title
+}) => {
+  return (
+    <StyledCard
+      className={`card ${className}`}
+      linkContainerMinHeight={linkContainerMinHeight}
+      linkUrl={linkUrl}
+      imageUrl={imageUrl}
+      footer={footer}
+      style={style}
+    >
+      <a href={linkUrl}>
+        {imageUrl && !icon
+          && (
+            <div
+              className={`card-image ${imageMargin && 'card-image--margin'}`}
+              style={{ backgroundImage: `url(${imageUrl})` }}
+            />
+          )}
+        {icon && !imageUrl
+          && (
+            <div className='card-icon'>
+              <Icon icon={icon} size={iconSize} />
+            </div>
+          )}
+      </a>
+      <div className='card-content'>
+        {title && <h5 className='card-title'>{title}</h5>}
+        {children}
+      </div>
+      {footer
+        && (
+          <div className='card-footer'>
+            {footer}
+          </div>
+        )}
```

Comment:
Nice structure with `card-content` and `card-footer` 👍  Does it make sense to also have a `card-header`? 🤔